src/declaration.d

```diff
@@ -1952,24 +1954,45 @@ public:
      */
     override final bool isDataseg()
     {
-        version (none)
+	    debug (LOG)
```

Comment:
please put back the version(none). Also, run `detab` and `tolf` on files before making PRs.